deeplake/util/connect_dataset.py

```diff
@@ -0,0 +1,103 @@
+from typing import Tuple, Optional
+
+from deeplake.client.client import DeepLakeBackendClient
+from deeplake.client.log import logger
+from deeplake.util.path import is_hub_cloud_path, get_path_type, get_org_id_and_ds_name
+from deeplake.util.exceptions import InvalidDestinationPathError, InvalidSourcePathError
+from deeplake.util.logging import log_visualizer_link
+
+
```

Comment:
some docstrings for the classes and methods here would be helpful

Reply:
Tried to make clear the purpose of those